sql/dolphinscheduler-postgre.sql

```diff
@@ -290,30 +290,150 @@ CREATE TABLE t_ds_error_command (
 DROP TABLE IF EXISTS t_ds_process_definition;
 CREATE TABLE t_ds_process_definition (
   id int NOT NULL  ,
+  code bigint NOT NULL,
   name varchar(255) DEFAULT NULL ,
   version int DEFAULT NULL ,
+  description text ,
+  project_code bigint DEFAULT NULL ,
   release_state int DEFAULT NULL ,
-  project_id int DEFAULT NULL ,
   user_id int DEFAULT NULL ,
-  process_definition_json text ,
-  description text ,
   global_params text ,
-  flag int DEFAULT NULL ,
   locations text ,
-  connects text ,
+  flag int DEFAULT NULL ,
   receivers text ,
   receivers_cc text ,
+  timeout int DEFAULT '0' ,
+  tenant_id int NOT NULL DEFAULT '-1' ,
   create_time timestamp DEFAULT NULL ,
+  update_time timestamp DEFAULT NULL ,
+  PRIMARY KEY (id),
+  CONSTRAINT process_definition_unique UNIQUE (name, project_id)
+) ;
+
+create index process_definition_index on t_ds_process_definition (project_code,id);
+
+DROP TABLE IF EXISTS t_ds_process_definition_log;
+CREATE TABLE t_ds_process_definition_log (
+  id int NOT NULL  ,
+  code bigint NOT NULL,
+  name varchar(255) DEFAULT NULL ,
+  version int DEFAULT NULL ,
+  description text ,
+  project_code bigint DEFAULT NULL ,
+  release_state int DEFAULT NULL ,
+  user_id int DEFAULT NULL ,
+  global_params text ,
+  locations text ,
+  flag int DEFAULT NULL ,
+  receivers text ,
+  receivers_cc text ,
   timeout int DEFAULT '0' ,
   tenant_id int NOT NULL DEFAULT '-1' ,
+  operator int DEFAULT NULL ,
+  operate_time timestamp DEFAULT NULL ,
+  create_time timestamp DEFAULT NULL ,
   update_time timestamp DEFAULT NULL ,
-  modify_by varchar(36) DEFAULT '' ,
-  resource_ids varchar(64),
   PRIMARY KEY (id),
   CONSTRAINT process_definition_unique UNIQUE (name, project_id)
 ) ;
 
-create index process_definition_index on t_ds_process_definition (project_id,id);
+create index process_definition_index on t_ds_process_definition_log (project_code,id);
```

Comment:
i think it should be different with "process_definition_index" in table "t_ds_process_definition_log".